project/src/main/java/com/google/sps/servlets/VolunteeringOpportunityDataServlet.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.servlets;
```

Comment:
Can you put this file in a different PR? I feel there are 2 features in the PR.

Reply:
Done - I have separated this PR into 3 PRs 1 per feature. Thanks.